features/location/impl/src/main/kotlin/io/element/android/features/location/impl/send/SendLocationNode.kt

```diff
@@ -14,24 +14,37 @@
  * limitations under the License.
  */
 
-package io.element.android.features.location.impl
+package io.element.android.features.location.impl.send
 
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import com.bumble.appyx.core.lifecycle.subscribe
 import com.bumble.appyx.core.modality.BuildContext
 import com.bumble.appyx.core.node.Node
 import com.bumble.appyx.core.plugin.Plugin
 import dagger.assisted.Assisted
 import dagger.assisted.AssistedInject
+import im.vector.app.features.analytics.plan.MobileScreen
 import io.element.android.anvilannotations.ContributesNode
 import io.element.android.libraries.di.RoomScope
+import io.element.android.services.analytics.api.AnalyticsService
 
 @ContributesNode(RoomScope::class)
 class SendLocationNode @AssistedInject constructor(
     @Assisted buildContext: BuildContext,
     @Assisted plugins: List<Plugin>,
     private val presenter: SendLocationPresenter,
+    analyticsService: AnalyticsService,
 ) : Node(buildContext, plugins = plugins) {
+
+    init {
```

Comment:
Nit: we should probably use `onBuilt` instead.

Reply:
I copy/pasted it from the rest of the codebase, perhaps we should make a global refactor of these kind of analytics?